resilience4j-framework-common/src/main/java/io/github/resilience4j/common/bulkhead/configuration/BulkheadConfigurationProperties.java

```diff
@@ -53,7 +53,7 @@ private BulkheadConfig buildConfigFromBaseConfig(InstanceProperties baseProperti
         CompositeCustomizer<BulkheadConfigCustomizer> compositeBulkheadCustomizer,
         String instanceName) {
         ConfigUtils.mergePropertiesIfAny(baseProperties, instanceProperties);
-        BulkheadConfig baseConfig = buildBulkheadConfig(BulkheadConfig.custom(), baseProperties,
+        BulkheadConfig baseConfig = createBulkheadConfig(baseProperties,
```

Comment:
this will not create a loop between createBulkheadConfig() and buildConfigFromBaseConfig() ? as inside buildConfigFromBaseConfig()  u call createBulkheadConfig() and inside createBulkheadConfig() u can  buildConfigFromBaseConfig() if there is base configuration ?

Reply:
Yes. There is a loop between `buildConfigFromBaseConfig()`and `createBulkheadConfig()`. 
Everytime we build BulkheadConfig, Need to check if there is `baseConfig`.

Reply:
Then when it will stop that loop of checks ?

Reply:
If there is no base config in instance properties, it will stop.